### PR TITLE
Houskeeping and polishing

### PR DESCRIPTION
**Pull request by MrTin**
Howdy! 🐮 👦 

You have done a great job so far so I wanted to contribute a little by fixing the things that got me stuck during setup. Here goes.

Highlights of the changes made:
- Introduction cleaned up and structurized.
- Added the information I got so far on logging.
- Elaborated on configuration and structured it a little.
- Created a page on User examples. I'm not sure if there are any but it would be a great way of showcasing what can be built with this library. I can always remove this if you like.

Let me know what you think

### CODE REVIEW

File: setup/configuration.md [hunk 1]
```diff
@@ -1,13 +1,33 @@
...
+the server.
+
+### Example for Windows:
+Set the environment by running `set NODE_ENV=development` before you start the
+server.
+
+## Overriding config using environment variables
+You can inject settings via environment variables in the `hive_` namespace. `_`
+is used as the hierarchy delimiter, e.g. to change the port the http server
+listens on you would set `HIVE_HTTP_PORT` to `1337`.
```
Comment (marcelklehr) [inline setup/configuration.md]: Note: the env vars are not case insensitive, keys must be specified with the exact same case as in the config files. Also worth noting is that the values are JSON.parsed, so a string would need quotes.

File: setup/configuration.md [hunk 2]
```diff
@@ -1,13 +1,33 @@
...
+
+### Logging (WIP)
+Logging is done using the `log4js-node` package. By default, everything logged
+is being redirected to `STDOUT`. If you need to connect your logger to another
+way, you can see all the different ways of doing the [Official node4js-node wiki page
```
Comment (marcelklehr) [inline setup/configuration.md]: There are a few words missing here, I think :)

File: README.md [hunk 3]
```diff
@@ -1,19 +1,42 @@
...
- * give guidance to new users of Hive.js (this is on the to-do list)
- * help with the setup of your own instance
- * show how to integrate Hive with your own app
- * familiarize developers with Hive's internals
+## What is Hive.js?
+Hive.js is a collaboration server and a web application that allows you to collaborate with others on any document in real-time. Examples of applications that could be build with Hive.js is [Google Docs](https://docs.google.com) and [Cloud 9 Code Editor (collaborative coding)](https://c9.io/).
 
-## Meta
-This book is a git repository [hosted on github](https://github.com/hivejs/docs). This means that anyone, including you, can edit it and submit the changes! It may, however, be more pleasant to read this book in a compiled form: either as a website at [docs.hivejs.org](http://docs.hivejs.org), or as an E-Book in ([PDF](https://www.gitbook.com/download/pdf/book/hivejs/hive-js-documentation), [ePUB](https://www.gitbook.com/download/epub/book/hivejs/hive-js-documentation) or [Mobi](https://www.gitbook.com/download/mobi/book/hivejs/hive-js-documentation)).
+See what the community have build by visiting the [User examples](user_examples/README.md) page.
```
Comment (marcelklehr) [inline README.md]: "built" ;)

**Comment by marcelklehr** [comment]
Other than the small issues: Looks good!

**Comment by MrTin** [comment]
Fixed the problems, let me know what you think! :)

**Comment by marcelklehr** [comment]
I've cherry-picked the major changes and left out the newline fix (the file is auto-generated by [gitbook](http://gitbook.com), which I use to edit the docs) and the user examples as there's nothing in there, yet. I'm not against including something like this in the docs.

**Pull request closed without merging**